rust/private/rustdoc_test.bzl

```
@@ -154,6 +171,14 @@ rust_doc_test = rule(
             providers = [rust_common.crate_info],
             mandatory = True,
         ),
+        "deps": attr.label_list(
+            doc = dedent("""\
+                List of other libraries to be linked to this library target.
+
+                These can be either other `rust_library` targets or `cc_library` targets if
+                linking a native library.
+            """),
```

I'm surprised the other attributes don't have a provider requirement for deps, can you add that here though since I don't think `rust_doc_test` would need any more?
```suggestion
            """),
            providers = [CrateInfo, CcInfo],
```
I think that syntax is correct but https://bazel.build/rules/lib/attr#label would have more info.

---

Done.